vmm/Cargo.toml

```diff
@@ -19,7 +19,7 @@ anyhow = "1.0"
 arch = { path = "../arch" }
```

Comment:
This commit message could do with a "why".

Reply:
I will update it!